provider.go

```diff
@@ -301,6 +294,13 @@ func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib ma
 	if tenantID == "" {
 		return fmt.Errorf("tenantId is not set")
 	}
+
+	azureCloudEnv, err := ParseAzureEnvironment(cloudName)
+	if err != nil {
+		log.Infof("cloudname is not valid")
```

Comment:
since we are returning the error here, we can remove this log

Reply:
Done